cloud/test/integration/controllermanager/controllermanager_suite_test.go

```diff
@@ -0,0 +1,66 @@
+package controllermanager
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
The test implementation of this pr do not use the feature of ginkgo/v2, it can seemlessly rollback to v1 through changing import package. So I think it will not interfere with #4398.